content/doc/tutorials/create-a-pipeline-in-blue-ocean.adoc

```diff
@@ -8,430 +8,294 @@ section: doc
 :toclevels: 3
 :imagesdir: ../../book/resources
 
-This tutorial shows you how to use the link:/doc/book/blueocean/[Blue Ocean]
-feature of Jenkins to create a Pipeline that will orchestrate building a simple
-application.
-
-Before starting this tutorial, it is recommended that you run through at least
-one of the initial set of tutorials from the link:..[Tutorials overview] page
-first to familiarize yourself with CI/CD concepts (relevant to a technology
-stack you're most familiar with) and how these concepts are implemented in
-Jenkins.
-
-This tutorial uses the same application that the
-link:../build-a-node-js-and-react-app-with-npm[Build a Node.js and React app
-with npm] tutorial is based on. Therefore, you'll be building the same
-application although this time, completely through Blue Ocean. Since Blue Ocean
-provides a simplified Git-handling experience, you'll be interacting directly
-with the repository on GitHub (as opposed to a local clone of this repository).
-
-*Duration:* This tutorial takes 20-40 minutes to complete (assuming you've
-already met the <<prerequisites,prerequisites>> below). The exact duration will
-depend on the speed of your machine and whether or not you've already
-<<run-jenkins-in-docker,run Jenkins in Docker>> from link:..[another tutorial].
+This tutorial shows you how to use the link:/doc/book/blueocean/[Blue Ocean] feature of Jenkins to create a Pipeline that will orchestrate building a simple application.
+
+Before starting this tutorial, be sure to run through at least one of the initial tutorials from the link:..[Tutorials overview] page to familiarize yourself with CI/CD concepts, relevant to the technology stack you're most familiar with, and how these concepts are implemented in Jenkins.
```

Comment:
```suggestion
Before you start this tutorial, be sure to run through at least one of the initial tutorials from the link:..[Tutorials overview] page. This will help you learn about CI/CD concepts relevant to the technology stack you're most familiar with and how these concepts are implemented in Jenkins.
```